symplyphysics/laws/dynamics/vector/acceleration_from_force.py

```diff
@@ -20,6 +37,34 @@ def force_law(acceleration_: Vector) -> Vector:
     return scale_vector(mass, acceleration_)
 
 
+# Derive this law from law of force and momentum
+# Condition: mass is constant
+
+time = force_momentum_law.time
+momentum_x = Function("momentum_x", units.mass * units.velocity)
```

Comment:
Use `units.momentum` or do not specify dimensions at all. We do not need them for proof and do not check them here.

Reply:
I only figured that might be good for documenting the code.

Reply:
It is but we do not verify them so it's easy to make a mistake in those dimensions, so it will make more harm than good for user.

Reply:
Alright, I see